tools/releaseBuild/Images/microsoft_powershell_ubuntu16.04/Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN echo SNAPCRAFT_URL: ${SNAPCRAFT_URL} && \
     git && \
   curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core' | jq '.download_url' -r) --output core.snap && \
   mkdir -p /snap/core && unsquashfs -d /snap/core/current core.snap && rm core.snap && \
```

Comment:
should this be removed?

Reply:
The 16.04 image is used for stable, which is still based one core.  The build will fail if we remove it.